s/snappy_java/build.sh

```diff
@@ -0,0 +1,46 @@
+# ----------------------------------------------------------------------------
+# Package       : lz4-java
+# Version       : master
+# Source repo   : https://github.com/xerial/snappy-java
+# Tested on     : RHEL_7.9
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Krishna Harsha Voora <krishvoor@in.ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given platform using
+#             the mentioned version of the package. It may not work as expected 
+#             with newer versions of the package and/or distribution.
+#             In such case, please contact "Maintainer" of this script.
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+# clone branch passed as argument, if none, use master
+if [ -z $1 ]
+then
+	BRANCH=""
+else
+	BRANCH="--branch $1"
+fi
+
+if [ "$BRANCH" == "" ]
+then
+	echo "BRANCH = master"
+else
+	echo "BRANCH = $BRANCH"
+fi
+
+git clone $BRANCH https://github.com/xerial/snappy-java || (echo "git clone failed"; exit $?)
+cd snappy-java
+
```

Comment:
Missed to run "**make**" to trigger snappy-java build ?

Reply:
Updated, thank you!